.eslintrc

```diff
@@ -1,8 +1,5 @@
 {
-  "env": {
-    "mocha": true,
-  },
-  "extends": "airbnb",
+  "extends": "amo",
```

Comment:
This is outside the scope of this PR, but we should rename this. I've filed: https://github.com/mozilla/eslint-config-amo/issues/7